pkg/apis/networking/types.go

```diff
@@ -36,6 +36,15 @@ type NetworkPolicy struct {
 	Spec NetworkPolicySpec
 }
 
+// Policy Type string describes the NetworkPolicy Rule type
+// This type is beta-level in 1.8
+type PolicyType string
+
+const (
+	PolicyTypeIngress PolicyType = "Ingress"
```

Comment:
I think each of these should have comments describing what they do.

e.g. "PolicyTypeIngress means a NetworkPolicy affects ingress traffic to selected pods"